### PR TITLE
Update readme and contribution guidelines in project

### PR DESCRIPTION
**Pull request by joesnellpdx**
### Description of the Change

Updating readme and doc files to provide more clear direction to contributing engineers. There is more needed in the long term, but this should help. 

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### Still needed:

- all legacy repos need to be marked as deprecated and link to this repository. 
- release guidelines
- publish to npm guidelines
- update doc site to use latest release guidelines

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ README.md</code></pre>
 
 ## Running Locally
 
+Naviate to the [docs](https://github.com/10up/component-library/tree/develop/docs) directory from the command line.
```
Comment (nicholasio) [inline docs/getting-started.md]:
```suggestion
Navigate to the [docs](https://github.com/10up/component-library/tree/develop/docs) directory from the command line.
```

File: packages/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Packages: Baseline Component Library
+
+Please read and follow [CONTRIBUTING](../CONTRIBUTING.md) guidelines when updating these components.
+
+The UI components and scripts that are distributed as an NPM package, and test/demo pages for each component.
+
+Each package contains a Readme file and `package.json` with instructions for working with that particular component.
+
+Engineers can work in that component and dest on components demo/index.html page or use the [e2e](../e2e) directory for local testing.
```
Comment (nicholasio) [inline packages/README.md]:
```suggestion
Engineers can work in that component and test on the components demo/index.html page or use the [e2e](../e2e) directory for local testing.
```

**Review by nicholasio — CHANGES_REQUESTED** [review]
Thanks for the updated, just found some typos

**Comment by joesnellpdx** [comment]
> Thanks for the updated, just found some typos

Typos fixed. GRR! Sorry.

**Review by nicholasio — APPROVED** [review]

**Pull request merged**